packages/backend/src/controllers/fanart.ts

```diff
@@ -0,0 +1,99 @@
+import { FanartDoc, FanartQuery } from '@model/fanart'
+import {
+  deleteFanart,
+  getAllFanarts,
+  storeFanart,
+  updateFanart,
+} from '@service/fanart'
+import {
+  responseBadRequest,
+  responseCreated,
+  responseInternalServerError,
+  responseSuccess,
+} from '@util/response'
+import { isBoolean, isString, isUndefined, isValidId } from '@util/validate'
+import { Request, Response } from 'express'
+
+export const getAllFanartsController = async (req: Request, res: Response) => {
+  try {
+    //  Gets [limit] fanart after _id [lastId]
+    const { lastId = 'NULL', limit = '10' }: FanartQuery = req.query
+    const fanarts: FanartDoc[] = await getAllFanarts(lastId, +limit)
+
+    return responseSuccess(res, fanarts)
+  } catch (error) {
+    return responseInternalServerError(res, error.message, error)
+  }
+}
+
+export const createFanartController = async (req: Request, res: Response) => {
+  try {
+    // Request body validation
+    const { creator, imageUrl } = req.body
+    if (!(creator && imageUrl))
+      throw new TypeError('creator and imageUrl is required')
```

Comment:
```suggestion
    if (!(creator && imageUrl)) throw new TypeError('creator and imageUrl is required')
```

Reply:
This is being auto-formatted by my IDE, so when I try to make it in oneline, my IDE shows an error